tests/unit/amo/reducers/test_collections.js

```diff
@@ -156,7 +156,7 @@ describe(__filename, () => {
     it('cannot load collection page without a current collection', () => {
```

Comment:
We don't generally write tests for our calls to `invariant`, so this test can be removed entirely.

Reply:
I think here is to test the case `without a current collection` and it doesn't use `invariant`, so maybe it is ok to keep?

Reply:
Right, I missed that. Thanks @xu3u4. 👍